ci-operator/step-registry/hypershift/mce/power/create/nodepool/hypershift-mce-power-create-nodepool-commands.sh

```diff
@@ -70,181 +65,38 @@ export IBMCLOUD_TRACE=true
+          inventory.agent-install.openshift.io/cpu-architecture: ppc64le
+    type: Agent
+  release:
+    image: ${OCP_IMAGE_MULTI}
+  replicas: 0
 EOF
 
-oc wait --timeout=5m --for=condition=DeploymentsHealthy agentserviceconfig agent
-echo "$(date) AgentServiceConfig ready"
-
-set +x
-# Setting up pull secret with brew token
-oc extract secret/pull-secret -n openshift-config --to=/tmp --confirm
-brewToken="${AGENT_POWER_CREDENTIALS}/brew-token"
-cat /tmp/.dockerconfigjson | jq --arg brew_token "$(cat ${brewToken})" '.auths += {"brew.registry.redhat.io": {"auth": $brew_token}}' > /tmp/pull-secret
-PULL_SECRET=/tmp/pull-secret
-set -x
-
-# Creating agent hosted cluster manifests
-echo "$(date) Creating agent hosted cluster manifests"
-oc create ns ${HOSTED_CONTROL_PLANE_NAMESPACE}
-mkdir /tmp/hc-manifests
-
-ICSP_COMMAND=""
-if [[ $ENABLE_ICSP == "true" ]]; then
-  ICSP_COMMAND=$(echo "--image-content-sources ${SHARED_DIR}/mgmt_icsp.yaml")
-fi
-
-${HYPERSHIFT_CLI_NAME} create cluster agent ${ICSP_COMMAND} \
-    --name=${HOSTED_CLUSTER_NAME} \
-    --namespace=${CLUSTERS_NAMESPACE} \
-    --pull-secret=${PULL_SECRET} \
-    --agent-namespace=${HOSTED_CONTROL_PLANE_NAMESPACE} \
-    --base-domain=${HYPERSHIFT_BASE_DOMAIN} \
-    --api-server-address=api.${HOSTED_CLUSTER_NAME}.${HYPERSHIFT_BASE_DOMAIN} \
-    --ssh-key=${SSH_PUB_KEY_FILE}\
-    --release-image=${OCP_IMAGE_MULTI} \
-    --control-plane-availability-policy=${CP_AVAILABILITY_POLICY} \
-    --infra-availability-policy ${HYPERSHIFT_INFRA_AVAILABILITY_POLICY} \
-    --render > /tmp/hc-manifests/cluster-agent.yaml
-
-# Split the manifest to replace routing strategy of various services
-csplit -f /tmp/hc-manifests/manifest_ -k /tmp/hc-manifests/cluster-agent.yaml /---/ "{6}"
-
-# Service strategy to replace
-printf "  - service: APIServer
-    servicePublishingStrategy:
-      type: LoadBalancer
-  - service: OAuthServer
-    servicePublishingStrategy:
-      type: Route
-  - service: OIDC
-    servicePublishingStrategy:
-      type: None
-  - service: Konnectivity
-    servicePublishingStrategy:
-      type: Route
-  - service: Ignition
-    servicePublishingStrategy:
-      type: Route
-  - service: OVNSbDb
-    servicePublishingStrategy:
-      type: Route
-" > /tmp/hc-manifests/replacement.yaml
-
-for file in /tmp/hc-manifests/manifest_*
-do
-    if grep -q 'kind: HostedCluster' "$file"
-    then
-        yq eval-all -i 'select(fileIndex==0).spec.services = select(fileIndex==1) | select(fileIndex==0)' "$file" "/tmp/hc-manifests/replacement.yaml"
-    fi
-done
-
-# Applying agent cluster manifests
-echo "$(date) Applying agent cluster manifests"
-ls /tmp/hc-manifests/manifest_* | awk ' { print " -f " $1 } ' | xargs oc apply
-
-oc wait --timeout=15m --for=condition=Available --namespace=${CLUSTERS_NAMESPACE} hostedcluster/${HOSTED_CLUSTER_NAME}
-echo "$(date) Agent cluster is available"
-
 # Applying InfraEnv
 echo "$(date) Applying InfraEnv"
 envsubst <<"EOF" | oc apply -f -
```

Comment:
Can we make the `InfraEnv` name Power specific?
Like, `${HOSTED_CLUSTER_NAME}-ppc64le`

Reply:
Done